README.md

```diff
@@ -263,6 +263,10 @@ Further Information
 | elasticsearch_data_stream_stats_json_parse_failures                  | counter    | 0           | Number of parsing failures for Data Stream stats                                                    |
 | elasticsearch_data_stream_backing_indices_total                      | gauge      | 1           | Number of backing indices for Data Stream                                                           |
 | elasticsearch_data_stream_store_size_bytes                           | gauge      | 1           | Current size of data stream backing indices in bytes                                                |
+| elasticsearch_cluster_license_expiry_date_seconds                    | gauge      | 1           | License expiry date since unix epoch in second s                                                    |
```

Comment:
```suggestion
| elasticsearch_cluster_license_expiry_date_seconds                    | gauge      | 1           | License expiry date since unix epoch in seconds                                                    |
```